<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HackFileWindow</class>
 <widget class="QDialog" name="HackFileWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>File name:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="fileNameLineEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Path to file:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="pathToFileLineEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Owner:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="ownerLineEdit">
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Owner's SID:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="ownerSidLineEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Integrity level:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="integrityLevelComboBox">
     <item>
      <property name="text">
       <string>Untrusted</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Low</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Medium</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>High</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Access Control List:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="aclTable">
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Holder</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Right</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
